bloom/generators/debian/templates/control.em

```diff
@@ -12,3 +12,10 @@ Depends: ${shlibs:Depends}, ${misc:Depends}, @(', '.join(Depends))
 @[if Conflicts]Conflicts: @(', '.join(Conflicts))@\n@[end if]@
 @[if Replaces]Replaces: @(', '.join(Replaces))@\n@[end if]@
 Description: @(Description)
+
+Package: @(Package)-dbg
+Architecture: any
+Section: debug
+Priority: extra
+Depends: @(Package) (= ${binary:Version}), ${shlibs:Depends}, ${misc:Depends}
```

Comment:
Any reason to include shlibs in Depends? [Debian documentation](https://wiki.debian.org/DebugPackage) does not include it, probably because they will be pulled form the `$(Package)`.